tests/check_framework/test_templates.py

```diff
@@ -0,0 +1,39 @@
+from django.core.checks.templates import E001
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+
+TEMPLATES = [
```

Comment:
I'd put this inside the class above the test and give it a name specific to the test in case more checks and tests are added.